cime_config/config_grids.xml

```diff
@@ -3322,6 +3352,8 @@
       <ny>1</ny>
       <file grid="atm|lnd" mask="oRRS18to6v3">$DIN_LOC_ROOT/share/domains/domain.lnd.ne256pg2_oRRS18to6v3.200212.nc</file>
       <file grid="ice|ocn" mask="oRRS18to6v3">$DIN_LOC_ROOT/share/domains/domain.ocn.ne256pg2_oRRS18to6v3.200212.nc</file>
+      <file grid="atm|lnd" mask="RRSwISC6to18E3r5">$DIN_LOC_ROOT/share/domains/domain.lnd.ne256pg2_RRSwISC6to18E3r5.250515.nc</file>
+      <file grid="ice|ocn" mask="RRSwISC6to18E3r5">$DIN_LOC_ROOT/share/domains/domain.ocn.ne256pg2_RRSwISC6to18E3r5.250515.nc</file>
```

Comment:
@wlin7 -- can I ask you how these domain files were generated? There is a known issue with gen_domain where it is truncating lnd fractions at a value that is too large by default

Reply:
Thanks for asking @jonbob . I used 

` gen_domain map_RRSwISC6to18E3r5_to_ne256pg2_traave.20250618.nc -o RRSwISC6to18E3r5 -l ne256pg2`

with gen_domain from the latest e3sm_unified. Are what documented  [here](https://e3sm.atlassian.net/wiki/spaces/COM/pages/4999905319/Atm-to-surface+traave+map+conservation+error) accounted for by the new version of gen_domain? I did assume that,

Reply:
I just replaced the domain files with ones generated using:
````
gen_domain -m /lcrc/group/e3sm/public_html/inputdata/cpl/gridmaps/RRSwISC6to18E3r5/map_RRSwISC6to18E3r5_to_ne256pg2_traave.20250618.nc -o RRSwISC6to18E3r5 -l ne256pg2 --fminval 1e-08
````
The number of lnd cells from the resulting domain file increased from 477527 to 478047